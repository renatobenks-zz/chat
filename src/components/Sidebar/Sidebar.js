// @flow

import styled from 'styled-components';

import { media } from '../../styles';

const Sidebar = styled.div`
  display: flex;
  flex-flow: column wrap;
  margin: -${props => props.theme.padding} 0 -${props => props.theme.padding} -${props => props.theme.padding};

  ${media.small`
    flex: 1;
  `};

  ${media.extraSmall`
    flex: 2;
  `};
`;

export default Sidebar;
