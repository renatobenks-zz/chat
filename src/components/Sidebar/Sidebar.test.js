import React from 'react';

import { render } from '../../../test/helpers';

import Sidebar from './Sidebar';

it('renders the sidebar', () => {
  expect(
    render(
      <Sidebar>
        <span>simple text</span>
      </Sidebar>,
    ),
  ).toMatchSnapshot();
});
