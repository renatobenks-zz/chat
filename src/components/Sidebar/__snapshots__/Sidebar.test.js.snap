// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the sidebar 1`] = `
<div
  className="sc-bdVaJa hLTngq"
>
  <span>
    simple text
  </span>
</div>
`;
